tests/test_flexmatch.py

```diff
@@ -353,6 +354,10 @@ def test_cannot_start_matchmaking_with_cancelling_ticket(self):
             self.delete(ticket_url, expected_status_code=http_client.OK)
             self.post(self.endpoints["flexmatch_tickets"], data={"matchmaker": "unittest"},
                       expected_status_code=http_client.CONFLICT)
+    def test_cannot_start_matchmaking_if_banned(self):
+        self._initiate_matchmaking()
+        with patch.object(flexmatch, 'is_player_banned', return_value=True):
+            self.post(self.endpoints["flexmatch_tickets"], data={"matchmaker": "unittest"}, expected_status_code=http_client.FORBIDDEN)
```

Comment:
Not sure how to test the Redis stuff so just retuning true to get the error here....